website/content/en/docs/overview/operator-capabilities.md

```diff
@@ -182,6 +183,9 @@ The RED Method defines the three key metrics for every service in your architect
 * Errors (the number of those requests that are failing)
 * Duration (the amount of time those requests take)
 
+Note that by building projects using Operator-SDK or [Kubebuilder][kubebuilder] CLI tools your solutions
+leverage on [controller-runtime][controller-runtime] which provides the above [metrics][metrics] exported by default. For further information, it's [reference][metric-reference] documentation to know what is exported by default. Also, you might want to give a look at the [(grafana/v1-alpha)][grafana-plugin-docs] which can help you out. 
```

Comment:
I believe you need to switch here.

Note that by building projects using Operator-SDK or [Kubebuilder][kubebuilder] CLI tools your solutions leverage on [controller-runtime][controller-runtime] which provides the following  [reference][metric-reference] exported by default. For further information, see the [metrics][metrics]  documentation to understand how to enable monitoring and add custom metrics . 
Also, you may want to give a look at the [(grafana/v1-alpha)][grafana-plugin-docs] which explains about setting up Grafana and provides a controller-runtime dashboard that you can import to it.